plotting/comparison.py

```diff
@@ -164,7 +164,11 @@
     if args.autorebin is not None:
         rebin_factor = int(args.autorebin)
         allaxis[collated[args.ref][discr].axes[-1].name] = hist.rebin(rebin_factor)
-    # xlabel = if  arg.xlabel is not None else collated["data"][discr].axes[-1].label # Use label from stored hists
+    xlabel = (
+        args.xlabel
```

Comment:
Yes, indeed. The intention was to leave it as a solution, in case someone wants to adjust the label for a dedicated plot (instead of using the labels from definitions.py). So either we put a hint there or remove it completely for now

Reply:
I see. then it would be fine to leave this current construction. maybe we can remove this for next round of commissioning